### DISCUSSION TITLE
Add Confirm and Cancel Buttons?

### DISCUSSION THREAD

**Discussion by the-dut** [henninghall/react-native-date-picker#270] [Q&A] [answered]
I'm not seeing any way to actually submit the date they pick. Is there supposed to be a confirm/cancel or select/cancel buttons that show up?  I'm not seeing any.  Thanks!

**Comment by henninghall** [comment]
Good question! it is up to you to implement that yourself, if you want that in your application. Since it is a common pattern I consider adding a default implementation of that in the future.

**Reply by OwaisNizami** [reply]
Dear Henninghall,
Did you add the confirm and cancel button in the date picker?

**Reply by henninghall** [reply]
Not yet, still in the backlog though!

**Comment by OwaisNizami** [comment]
What do you mean by still in backlog though?
I want to implement an IOS type date picker in android. Please confirm if
there is any other library?


*Regards,Muhammad Owais Nizami <https://www.facebook.com/owais.nizami>*
*+92 322 45 09 676*

**Comment by henninghall** [comment, marked as answer]
This feature is now implemented and released in `v4.0.0`

**Reply by OwaisNizami** [reply]
that's great 👍
